mlxtend/feature_selection/sequential_feature_selector.py

```diff
@@ -399,7 +449,7 @@ def fit(self, X, y, custom_feature_names=None, groups=None, **fit_params):
                 k_idx = self.fixed_features_
                 k = len(k_idx)
                 k_idx, k_score = _calc_score(self, X_, y, k_idx,
-                                             groups=groups, **fit_params)
+                                             groups=groups, weights=self.sample_weights, **fit_params)
```

Comment:
See my comment above. I don't think you need `weights` here since you have `self`.